src/style/disease-table.module.css

```diff
@@ -44,15 +44,24 @@
     display: none; /* remove dividers between column headers */
 }
 
-.container .hover-column:global(.ant-table-cell):has(a):hover {
+.container .data-complete .hovered-row:global(.ant-table-cell) {
```

Comment:
do you need `:global(.ant-table-cell)` here? it seems like 3 class names should be specific enough, and if possible it's nice to not have to use the global selector

Reply:
It's specific enough for the pointer but not the background-color, I lose that style if I remove it.